content/news/public-oss-demo/2024/upcoming-live-update-october-2024.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "Announcing PrestaShop Project Live Update October 2024"
+subtitle: "Join us on YouTube on October 30th, 2024 at 4 pm (Paris time) for the next Live Update!"
+date:   2024-10-29
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community, live-update]
+---
+
+It's been a while but the next Live Update for the PrestaShop Project in 2024 will be back on **Wednesday, October 30th, 2024** at [4 pm (Paris time)](https://time.is/1600_30_Oct_2024_in_Paris) on [YouTube](https://www.youtube.com/watch?v=BfQzUWhQJCE)!
+
+In the upcoming Live Update, we will provide updates on the latest developments in the PrestaShop Project, including PrestaShop 8.2 version 9 and more. It will also be a great occasion to discuss the PrestaShop Developer Conference 2024 in detail and tell you more about our plans for the future.
```

Comment:
```suggestion
In the upcoming Live Update, we will provide updates on the latest developments in the PrestaShop Project, including PrestaShop 8.2, version 9 and more. It will also be a great occasion to discuss the PrestaShop Developer Conference 2024 in detail and tell you more about our plans for the future.
```